### PR TITLE
Groovy scripts for AutoTest Jobs

### PR DESCRIPTION
**Pull request by d-jana**
##### SUMMARY
Groovy scripts for AutoTest Jenkins Build. This also updated city and salesforce.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
Groovy scripts for AutoTest which calls from Jenkins build and used to provision environments on CFME

##### ADDITIONAL INFORMATION
This also updated city=jenkins and salesforce=gptejen so that it will be easy way to identify our workloads and track the cost.

### CODE REVIEW

File: tests/jenkins/ansible-advanced-homework.groovy [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+        stage('order from CF') {
+            environment {
+                uri = "${cf_uri}"
+                credentials = credentials("${opentlc_creds}")
+                DEBUG = 'true'
+            }
+            /* This step use the order_svc_guid.sh script to order
+             a service from CloudForms */
+            steps {
+                git url: 'https://github.com/fridim/cloudforms-oob'
```
Comment (fridim) [inline tests/jenkins/ansible-advanced-homework.groovy]: you can use the new URL
https://github.com/redhat-gpte-devopsautomation/cloudforms-oob

File: tests/jenkins/ansible-advanced-three-tier-app.groovy [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+                          ./tests/jenkins/downstream/poll_email.py \
+                          --server '${imap_server}' \
+                          --guid ${guid} \
+                          --timeout 40 \
+                          --filter 'has completed'
+                        """
+                    ).trim()
+
+                    try {
+                        def m = email =~ /ssh -i ~/.ssh/your_private_key_name ([^ \n]+)/
```
Comment (fridim) [inline tests/jenkins/ansible-advanced-three-tier-app.groovy]: Is this regex working ?
You don't have to escape the `/` ?

```groovy
                        def m = email =~ /ssh -i ~\/.ssh\/your_private_key_name ([^ \n]+)/
```

File: tests/jenkins/ansible-advanced-homework.groovy (hunk 1 above)
Comment (d-jana) [reply tests/jenkins/ansible-advanced-homework.groovy]: Fixed [3fe6967](https://github.com/redhat-cop/agnosticd/pull/939/commits/3fe696786147d16fedb1c5564337d249fa17db58)

File: tests/jenkins/ansible-advanced-homework.groovy [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+                          ./tests/jenkins/downstream/poll_email.py \
+                          --server '${imap_server}' \
+                          --guid ${guid} \
+                          --timeout 40 \
+                          --filter 'has completed'
+                        """
+                    ).trim()
+
+                    try {
+                        def m = email =~ /ssh -i ~/.ssh/your_private_key_name ([^ \n]+)/
```
[the remark this answers is not in the document]
Comment (d-jana) [reply tests/jenkins/ansible-advanced-homework.groovy]: fixed.

File: tests/jenkins/ansible-advanced-three-tier-app.groovy (hunk 2 above)
Comment (d-jana) [reply tests/jenkins/ansible-advanced-three-tier-app.groovy]: fixed.

**Pull request merged**